### ISSUE TITLE
Flyer for motivating the use of the CWA for check-ins.

### ISSUE THREAD

**Issue by alanrick** [corona-warn-app/cwa-wishlist#519]
## Where to find the issue
There is a lack of CWA check-in QR-Codes printed in places where it could be useful, such as meeting rooms or shops.

## Describe the issue
Having spoken to acquaintances and store-owners I believe there is a confusion about which check-in mechanism to use, why it is offered by the CWA, and indeed a lack of awareness that it is offered at all.

## Suggested change
I have been given the opportunity locally to distribute a flyer to the local business community. Whether or not it is approved for distribution is yet to be decided, but I'm posting it here in the hopes that I receive feedback, such as corrections.

If it proves useful, I'm happy uploading the raw document for use in other Bundesländer.

[CoronaDigitalRegistrierung.pdf](https://github.com/corona-warn-app/cwa-documentation/files/6461325/CoronaDigitalRegistrierung.pdf)

**Comment by heinezen** [comment]
@alanrick 

I've moved your post to the wishlist. Seems like a better place for discussion. The community should feel fre to give feedback.

----
Corona-Warn-App Open Source Team

**Comment by alanrick** [comment]
Thank you @heinezen . I've updated it with minor corrections.

**Comment by vaubaehn** [comment]
I really appreciate the effort of @alanrick !

Since May 13, the situation has changed. Incidence rates are currently low, and I would not be surprised when at least some Federal states will cancel anti-COVID-measures as a whole, including registering personal contacts via Luca or paper lists.

However, in my opinion we're probably not done yet for a possibly long time:
- New, highly infectious variants are on their way
- Many people do not want to get vaccinated, few can't for health reasons
- We don't know yet for how long vaccinations will protect us, at what times boosters are needed.

But we already have nice features built into CWA, like the check-in, that we may continue to use and by this decrease the r-rate probably a little.

Still many places are not aware of the usefulness of the event check-in feature, how easy it is to use with nearly no effort for the QR code provider.

I wonder if the CWA user community could make a change, when trying to engage other users via social media, to ask their places (shops, events, doctors, etc.) to provide CWA check-in codes?
@Ein-Tim for example already has good connections via Twitter...

What are your opinions?

**Comment by Ein-Tim** [comment]
@vaubaehn 

I agree with you that the CWA check-in features has not enough attention, but in the most cases it would just be illegal for the venues to offer to guests to either check in with CWA **or** another app/paper list. AFAIK only Lower Saxony currently really allows to only use the CWA. 

And, there are many questions how cross-system warnings work (or don't work), see #486.

**Comment by vaubaehn** [comment]
@Ein-Tim 
> but in the most cases it would just be illegal for the venues to offer to guests to either check in with CWA **or** another app/paper list.

Right, but my thoughts were going into the future when paper lists and Luca won't be mandatory anymore. In that case CWA could nicely replace current solutions (personal tracking) instead of complementing them. That would need be completely voluntarily then, for users and venue owners.
For sure, already in current days CWA's check-in could complement personal tracking solutions much better, but unfortunately most venues don't understand why it's a good idea to provide CWA additionally. Hence the idea to create more publicity originating from user community.

> And, there are many questions how cross-system warnings work (or don't work), see #486.

Same applies here, imho.

**Comment by Ein-Tim** [comment]
@vaubaehn 

Yes, I agree with you, in any case there should be more attention to the Corona-Warn-App Check-In Feature!
I'll think about how we could improve this!

Have a good night! 😴

**Comment by alanrick** [comment]
Thanks for the encouragement @vaubaehn .

I can only offer anecdotal feedback from one Gemeinde.
1. I updated the brochure with the (then) current covid BaWü rules and  the Oberbürgermeister distributed it to the local businesses. I didn't get any feedback about the content other than personal confirmation by acquaintances running businesses that it was read.
2. The feedback from the local Gesundheitsamt (before publishing) was that they much prefer dealing with the track&tracing themselves (e.g. from paper/electronic lists) rather than relying on the visitors taking responsibility (CWA). 
3. Most local bars/restaurants offer paper-only. Next come those offering Luca+paper and some local businesses offer a choice between Luca and CWA. This is not _illegal_, but it is not correct either, and does risk attention from the Ordnungsamt.  One business offered a home-coded electronic solution.
4. No local establishments offer a distinction between the different seating areas. This is worrying. If the Gesundheitsamt is very busy and doesn't have the staff to follow up individually then all visitors for that period may be sent into quarantine. This makes little sense for an outdoor venue where the risk of aerosol/contact is low but mass-quarantining is a real possibility.
5. None of the establishments verify that the visitors register with their real name (in BaWü it's not their responsibility) or really scan the qr-codes. I heard of visitors  slotting blank slips of paper into the collection box or pretending to scan a qr-code. Trust in the Gesundheitsamts seems low (again anecdotal evidence  - people being sent into quarantine long after the risk period was over)
6. There was very little awareness of the distinction between aerosol and direct transmission. No distinction between seating outdoors and indoors. No awareness of the need to check-in for meeting rooms compared to entering a large shop.

My personal conclusion was that the CWA bluetooth monitoring is extremely useful, but the complexity of the aerosol-scenario made check-ins (CWA or Luca) irrelevant, and perhaps even counter-productive, other than the personal history protocol (depending on app settings).

I remain a strong supporter for the CWA check-in feature to indirectly propagate the bluetooth proximity tracking, but the Corona Verordnungen need updating to allow CWA check-ins. If alternative check-ins apps are allowed (e..g Luca) then Corona Verordnungen need to be radically updated to take into account that the venue owners don't have the scientific judgement necessary to support check-in systems integrated with the Gesundheitsämter (e.g. the Corona Verordnung could make check-ins mandatory in meeting rooms but not in beer gardens).

**Comment by vaubaehn** [comment]
Hi @alanrick ,

>     1. I updated the brochure with the (then) current covid BaWü rules and  the Oberbürgermeister distributed it to the local businesses. I didn't get any feedback about the content other than personal confirmation by acquaintances running businesses that it was read.

That sounds like a good idea to go via the mayor. I was thinking about doing similar with our Bürgermeister of our small town.
And a feedback that it was read at least, is already better than nothing!

>     2. The feedback from the local Gesundheitsamt (before publishing) was that they much prefer dealing with the track&tracing themselves (e.g. from paper/electronic lists) rather than relying on the visitors taking responsibility (CWA).

I can imagine. My hope is, that when incidence rates are more low in general and vaccination has a bigger impact, anonymous tracing via BLE and the here discussed Event Registration becomes more important (as paper lists/personal registration via app may be quit), eventhough it would be a volunteerly approach then.

>     3. Most local bars/restaurants offer paper-only. Next come those offering Luca+paper and some local businesses offer a choice between Luca and CWA. This is not _illegal_, but it is not correct either, and does risk attention from the Ordnungsamt.  One business offered a home-coded electronic solution.

That's how I also perceived it from my environment.

>     4. No local establishments offer a distinction between the different seating areas. This is worrying. If the Gesundheitsamt is very busy and doesn't have the staff to follow up individually then all visitors for that period may be sent into quarantine. This makes little sense for an outdoor venue where the risk of aerosol/contact is low but mass-quarantining is a real possibility.

I agree in general. For my personal security feelings, I'd prefer an "all visitors of a given time period need to take at least a test"-solution for indoors, and a more relaxed approach for outdoors.

>     5. None of the establishments verify that the visitors register with their real name (in BaWü it's not their responsibility) or really scan the qr-codes. I heard of visitors  slotting blank slips of paper into the collection box or pretending to scan a qr-code. Trust in the Gesundheitsamts seems low (again anecdotal evidence  - people being sent into quarantine long after the risk period was over)

Hm, from the perspective of solidarity, such behaviours are sad and poor in my personal opinion. On the other hand, this underlines that a volunteer approach using CWA Event Registration is at least similar successful, if not even better.

>     6. There was very little awareness of the distinction between aerosol and direct transmission. No distinction between seating outdoors and indoors. No awareness of the need to check-in for meeting rooms compared to entering a large shop.

*sigh*

> My personal conclusion was that the CWA bluetooth monitoring is extremely useful, but the complexity of the aerosol-scenario made check-ins (CWA or Luca) irrelevant, and perhaps even counter-productive, other than the personal history protocol (depending on app settings).

I'm disagreeing in this point, that Event Registration per se is irrelevant or counter-productive. Personally I think, that Event Registration could still be very useful, given venue owners are informed well-enough on how to apply it correctly and users are informed and motivated enough to use it accordingly. Especially as some new virus variants have the potential to spread even easier and faster, the aerosol-scenario becomes even more important. Due to my personal constitution I'm not so sure how protective my vaccination will be, and I'd like to get warned after visiting a restaurant (or similar) to protect my beloved ones that also have a higher risk to get infected and get the disease eventhough they're vaccinated. CWA Event Registration is no guarantee at all here that this happens, but if used, it at least increases the likelihood.

> I remain a strong supporter for the CWA check-in feature to indirectly propagate the bluetooth proximity tracking, but the Corona Verordnungen need updating to allow CWA check-ins. If alternative check-ins apps are allowed (e..g Luca) then Corona Verordnungen need to be radically updated to take into account that the venue owners don't have the scientific judgement necessary to support check-in systems integrated with the Gesundheitsämter (e.g. the Corona Verordnung could make check-ins mandatory in meeting rooms but not in beer gardens).

I fully agree. The Corona Verordnungen could do a lot here: when the Federal States see themselves at a point one day, to quit either on all measure, or to at least leave some measures still in effect, CWA Event Registration could be one of the measures that does not cost that much like luca, which is based on being used volunteerly by the users (whilst the venue owner would need to provide it) and is only weakly interfering into the personal space and freedom (similarly like wearing masks, even less imho).

Looks like, it would be good to increase awareness via local communities (Bürgermeister and co), health ministries of the Federal States, and randomly by talking to venue owners that we're going to visit.

**Comment by Ein-Tim** [comment]
BTW, I got [feedback](https://twitter.com/RobertKoeHH/status/1414534318907662342) from someone on Twitter, who told me that some venues decided against the Corona-Warn-App event-registration feature because they thought that they have to renew the QR-Code every day. 
The communication reg. this could also be improved. A QR-Code should (for security reasons) be renewed every day but it's not mandatory.

**Comment by alanrick** [comment]
> BTW, I got feedback …  that some venues decided against the Corona-Warn-App event-registration feature because they thought that they have to renew the QR-Code every day 

Which is why the brochure had a less alarming statement in the qrcode section independent of CWA:
 _In regelmäßigen Abständen (empfohlen wird einmal pro Woche) einen neuen QR-Code ausdrucken, um Missbrauch zu verhindern._

I notice the[ CWA generator](https://www.coronawarn.app/de/eventregistration/) web page has no such statement either before or after generating. Not sure where the “every day” recommendation comes from. 

I hear from the UK (where the incidence is rising) that more and more users are deinstalling the NHS equivalent to Luca because they do not want to receive you-must-quarantine messages from the health authorities. Neither the UK nor RKI tracks App usage figures (only downloads) so there’s a risk they’re out of touch. 
Decentralised CWA is the way to go 👍

**Comment by jucktnich** [comment]
@Ein-Tim The same does apply to Luca, it's the same security risk for both apps if you don't change the qrcodes.

**Comment by Ein-Tim** [comment]
@jucktnich 

Does the luca app mention to renew the QR-Codes? I don't think so. 
CWA does mention it and I assume that some users misunderstand the text and think they have to.

**Comment by jucktnich** [comment]
@Ein-Tim I thought it was added somewhere after Jan Böhmermann, but I'm not sure.

**Comment by Ein-Tim** [comment]
@jucktnich Anyways, as you can read in the Twitter Thread I shared above, the user only talks about CWA, not about luca.

**Comment by jucktnich** [comment]
Maybe add an FAQ article explaining the problem

**Comment by Ein-Tim** [comment]
@jucktnich And another report: https://twitter.com/jaeger1972/status/1405169435288211464?s=20

I'll think about a FAQ entry for this.

**Comment by alanrick** [comment]
Is the referenced screenshot from the App itself? This is what I see on mine (iOS) and I think the statement is quite reasonable and not demotivating without requiring an FAQ elsewhere.
 ![image](https://user-images.githubusercontent.com/7034358/125942927-5f20325e-1234-4c9f-a78e-46b4ebc29b79.jpeg)

**Comment by Ein-Tim** [comment]
@alanrick 

It's the paragraph above that causes confusion:

<img src="https://user-images.githubusercontent.com/67682506/125943478-8066d9f7-84c8-4088-ad89-d08abac79d98.jpeg" width="300">

**Comment by alanrick** [comment]
@Ein-Tim what's the source?

**Comment by Ein-Tim** [comment]
@alanrick 

It's inside of CWA. 

Here's a screen recoding: 

https://user-images.githubusercontent.com/67682506/125950535-8745f6ca-d802-4bc3-871a-79b65b09d772.MOV

**Comment by alanrick** [comment]
Aha thanks. I'd missed that. It sort of contradicts the information in the Data Protection and Data Security section where I'd made my screenshot. I much prefer _regular_ as opposed to _once a day_ since one size doesn't fit all. 

If I owned a bar I'd create location with a description "AlanRick's Watering hole - please let us know if you receive covid alerts" and wouldn't regenerate more than once a month unless I heard of suspicious behaviour.

**Comment by Ein-Tim** [comment]
@alanrick Yes, I agree with you. Instead of taking care of a new FAQ entry, I'll open an issue requesting to improve this text.

Sorry for highjacking this issue. 😅

**Comment by alanrick** [comment]
All in a good cause, @Ein-Tim :)  
But even this aspect - the evil spoofing of qr-codes - highlights another advantage of CWA over Luca et al, since motivation will be lower to spoof because the false reports don't go directly to the Gesundheitsamt.

**Comment by alanrick** [comment]
The latest Baden Württemberg Corona [Verordnung ](https://www.baden-wuerttemberg.de/fileadmin/redaktion/dateien/PDF/Coronainfos/211013_CoronaVO_konsolidierte_Fassung_ab_211015.pdf)apparently allows the Corona Warnapp to be used instead of paper lists/Luca et al. This is not mentioned directly in the Corona Verordnung, but is mentioned in text on the [official BW Web site](https://www.baden-wuerttemberg.de/en/service/aktuelle-infos-zu-corona/faq-corona-verordnung/)
![CoronaWarnAppDeltaBaWüCV](https://user-images.githubusercontent.com/7034358/138131929-5ed40290-4402-411b-91ce-8bd0a52111a7.png)

If true, this is something to be celebrated, and since the BW minister president has stated his intention to keep a Corona Verordnung in place in the foreseeable future I was thinking of updating the flyer.

However,  examining the text of the Verordnung I'm not sure that this is true, and not sure what it implies.
§8 (1) still states that the name and contact details of visitors are to be collected. That paragraph has not changed at all from the previous version that was not satisfied by the CWA. The only change made is that in section §8 (4), deleting the directive that the visitor's data needs to be transferrable in a safe and readable manner to the health/trace Center. 

There is no mention of the check-in function anywhere. And there is no mention that §8 (1) has been dropped. 

So I'm wondering:
a) Whether the CWA can really be used, because it does not fulfil §8 (1) since the visitor's name is not stored in the app or stored anywhere when joining an event.

And at the other extreme...
b) Whether an event needs to offer a CWA check-in qr-code because the user can do this her/himself in the app (fulfilling the requirements of §8 (4), and perhaps even just having the CWA active is sufficient because that on its own exchanges data between visitors, and might meet §8 (4) requirements on its own.

I'm reluctant to update the flyer without the confidence that the CWA does definitely satisfy the Verordnung so I'd be grateful if someone could explain how the CWA (a) satisfies the Verordnung and (b) whether the event must display a check-in qr-code.


---------------------
The markup shows the change between the September and October Verordnung.

 
![CoronaVOBaWüDeltaSeptOkt](https://user-images.githubusercontent.com/7034358/138137554-4bc8115a-3696-4758-a7f1-9aa9c27326aa.png)

**Comment by jucktnich** [comment]
For example in NRW its now even illegal to use Luca.

**Comment by alanrick** [comment]
Or option c) a slightly hidden option to create your own check-in. 
That's what I discovered today and the restaurant accepted it.

**Comment by alanrick** [comment]
In the [official justification](https://www.baden-wuerttemberg.de/fileadmin/redaktion/dateien/PDF/Coronainfos/211013_Begruendung_zur_1.AenderungsVO_zur_11.CoronaVO.pdf) for the rules, I found the following:
`Zu § 8 (Datenverarbeitung)
Zu Absatz 4
Durch die Streichung des 2. Halbsatzes von Satz 1 erfolgt eine Anpassung an die
geänderte Nachverfolgungspraxis. Es können fortan entsprechend weitere
wirkungsvolle digitale Applikationen, wie z.B. die Corona-Warn-App für die
Kontaktnachverfolgung genutzt werden. Aufgrund der zuletzt stabilen Infektionslage
ist es unter Verhältnismäßigkeitsgesichtspunkten geboten, auch solche Applikationen
zuzulassen, welche keine direkte Nachverfolgung durch das Gesundheitsamt
ermöglichen.`

Taking into account _Verhältnismäßigkeitsgesichtspunkten_, and _keine direkte Nachverfolgung durch das Gesundheitsamt_ I assume that even though the _Verordnung_ itself appears to reject the use of the Corona Warn App, **this is an error rather than an intention**. So bearing in mind there are so many downloads of the CWA (the closest the RKI has managed to get to estimating the active coverage of the CWA), and the focus on _Verhältnismäßigkeitsgesichtspunkten_, I'd assume that CWA alerting is reasonably effective through bluetooth or registration (_indirectly_ from the health-centers after positive tests).

So I come to the conclusion that the intention is that a) b) or c) are sufficient to satisfy §8. I.e. Having the CWA active on your phone is all you need to be comply with the law. Ditto those checking. I will try to get this confirmed by the local Ordnungsamt.

**Comment by alanrick** [comment]
I contacted the BW hotline (0711 90439555) and was told that each of the alternatives a), b), or c) is acceptable. Great news. 🎉🥳🎉

This is a real incentive to download the CWA app, although I'd advise venues to display the QR code to remove any ambiguity.
